kani-driver/src/assess/args.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This module contains arguments specific to the `cargo kani assess` subcommand.
+
+use std::path::PathBuf;
+
+use clap::Parser;
+
+/// `cargo kani assess` subcommand arguments
+#[derive(Default, Debug, Parser)]
+pub struct AssessArgs {
+    #[command(subcommand)]
+    pub command: Option<AssessSubcommand>,
+
+    /// Write Assess metadata (unstable file format) to the given file
+    #[arg(long, hide = true)]
+    pub emit_metadata: Option<PathBuf>,
+}
+
+/// `cargo kani assess` takes optional subcommands to request specialized behavior
+#[derive(Debug, Parser)]
+pub enum AssessSubcommand {
+    /// Run assess on a directory containing multiple cargo projects, and aggregate the results
+    Scan(ScanArgs),
+}
+
+/// `cargo kani assess scan` subcommand arguments
+#[derive(Debug, Parser)]
+pub struct ScanArgs {
+    /// Don't run assess on found packages, just re-analyze the results from a previous run
+    #[arg(long, hide = true)]
+    pub existing_only: bool,
```

Comment:
I bikeshedded on this name a bit already, I don't expect this is one a customer will want to use (hence `hide = true`). It's mostly useful while debugging the package-finding code, or table-rendering code.

I settled on 'existing' because I wanted to emphasize that something wouldn't be re-run here, while `analyze` is... what assess does already really. A good short name is hard to find.